clang/test/CIR/CodeGen/call.cpp

```diff
@@ -11,4 +11,4 @@ int f() {
 // CHECK:   %1 = cir.call @_Z1pv() : () -> !cir.ptr<!s32i>
 // CHECK:   %2 = cir.load %1 : cir.ptr <!s32i>, !s32i
 // CHECK:   %3 = cir.const(#cir.int<22> : !s32i) : !s32i
-// CHECK:   %4 = cir.binop(sub, %2, %3) : !s32i
+// CHECK:   %4 = cir.binop(sub, %2, %3) nsw : !s32i
```

Comment:
Using my suggestion, this would become `%4 = cir.binop(sub, %2, %3) wrap : !s32i`